DiscordBot.Modules/CommandModules/ReactionModule.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Discord;
+using Discord.Commands;
+using Microsoft.Extensions.Logging;
+
+namespace DiscordBot.Modules.CommandModules
+{
+    public class ReactionModule : ModuleBase
+    {
+        private readonly ILogger<PingModule> _logger;
+
+        public ReactionModule(ILogger<PingModule> logger)
+        {
+            _logger = logger;
+        }
+
+        [Command("react")]
+        [Summary("Der Bot reagiert mit dem angegebenen Emoji auf die angegebene Nachricht.")]
+        public async Task AddReaction(string name, string URL)
+        {
+            await ReactWithEmoji(name, URL);
+        }
+        private async Task ReactWithEmoji(string name, string URL)
```

Comment:
You could write this shorter and just add the command and summary attribute to this method.

Reply:
???

Reply:
You call this method in the method above, without any else code and the same parameters. Just remove the lines 24-26 😀

Reply:
Thats exactly what you did, I copyed from you 😁

Reply:
Yeah, i forgot that 😉

Reply:
And what I needed the method for multiple commands 😀